specification/_global/msearch/MultiSearchRequest.ts

```diff
@@ -101,13 +101,14 @@ export interface Request extends RequestBase {
     include_named_queries_score?: boolean
     /**
      * Maximum number of concurrent searches the multi search API can execute.
+     * Defaults to `max(1, (# of data nodes * min(search thread pool size, 10)))`.
      */
-    max_concurrent_searches?: long
+    max_concurrent_searches?: integer
```

Comment:
This is also an integer: https://github.com/elastic/elasticsearch/blob/36c14bf3a56f85177f5ce60b2f839b6b35236bdc/server/src/main/java/org/elasticsearch/rest/action/search/RestMultiSearchAction.java#L115-L117